File: python/data_structures/stack.py
'''
Lists in Python doubles down as stacks and queues.
'''
stack = []
stack.append(1)
stack.append(2)
stack.append(3)
print(stack)  # [1, 2, 3]
print(stack.pop())  # 3
print(stack)  # [1, 2]
print(stack[-1]) # Top element of the stack

File: js/data_structures/stack.js
const stack = [];

stack.push(1);
stack.push("focus");
stack.push(true);
stack.push(null);
stack.push(undefined);
stack.push({ name: "John" });
stack.push([1, 2, 3]);
stack.push(function () {
  console.log("This is a function");
});

console.log(stack);

console.log(stack[stack.length - 1]);

console.log(stack.pop());
console.log(stack);
